integration/spark/Dockerfile

```diff
@@ -17,7 +17,7 @@
 FROM arrow:python-3.6
 
 # installing java and maven
-ARG MAVEN_VERSION=3.5.4
+ARG MAVEN_VERSION=3.6.2
```

Comment:
This is the minimum version used by Spark, so setting this here will prevent Spark from downloading it during the build phase